neard/src/cli.rs

```diff
@@ -50,7 +50,7 @@ impl NeardCmd {
 
             NeardSubCommand::UnsafeResetData => {
                 let store_path = get_store_path(&home_dir);
-                info!(target: "neard", "Removing all data from {}", store_path);
+                info!(target: "neard", "Removing all data from {}", store_path.to_str().unwrap());
```

Comment:
```suggestion
                info!(target: "neard", "Removing all data from {}", store_path.display());
```

again, there are unwrap-free APIs for working with paths

Reply:
fixed